src/PeriodEnd/TransferPayments/SFA.DAS.ProviderPayments.Calc.TransferPayments.UnitTests/ServiceTests/WhenProcessingSendingAccounts.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using AutoFixture.NUnit3;
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dal.DatabaseEntities;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Dependencies;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Domain;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.Services;
+using SFA.DAS.ProviderPayments.Calc.TransferPayments.UnitTests.Utilities;
+
+namespace SFA.DAS.ProviderPayments.Calc.TransferPayments.UnitTests.ServiceTests
+{
+    [TestFixture]
+    public class WhenProcessingSendingAccounts
+    {
+        [Test, TransfersAutoData]
```

Comment:
Does this test put the items into the wrong order to begin with?
Same for test below.